crates/types/src/vid.rs

```diff
@@ -66,6 +67,18 @@ pub fn vid_scheme(num_storage_nodes: usize) -> VidSchemeType {
     VidSchemeType(Advz::new(num_storage_nodes, recovery_threshold, &*KZG_SRS).unwrap_or_else(|err| panic!("advz construction failure:\n\t(num_storage nodes,recovery_threshold)=({num_storage_nodes},{recovery_threshold})\n\terror: : {err}")))
 }
 
+/// Similar to [`vid_scheme()`], but with `KZG_SRS_TEST` for testing purpose only.
+#[cfg(feature = "test-srs")]
+pub fn vid_scheme_for_test(num_storage_nodes: usize) -> VidSchemeType {
+    let recovery_threshold = 1 << num_storage_nodes.ilog2();
+    #[allow(clippy::panic)]
+    let num_storage_nodes = u32::try_from(num_storage_nodes).unwrap_or_else(|err| {
+        panic!("num_storage_nodes {num_storage_nodes} should fit into u32\n\terror: : {err}")
+    });
+    #[allow(clippy::panic)]
+    VidSchemeType(Advz::new(num_storage_nodes, recovery_threshold, &*KZG_SRS_TEST).unwrap_or_else(|err| panic!("advz construction failure:\n\t(num_storage nodes,recovery_threshold)=({num_storage_nodes},{recovery_threshold})\n\terror: : {err}")))
```

Comment:
Since these lines are touched, I want to bring this up: can we remove the newlines and tabs from these logs? It creates non-standard outputs and can cause problems in a structured context.

Reply:
no objection so long as Alex is the one to do it. 😆

Reply:
what new lines and tabs? you mean line 76?